doc/whats_new.rst

```diff
@@ -12,6 +12,9 @@ Current
 
 Changelog
 ~~~~~~~~~
+- Ability to select which drives to optimize for event related potentials. 
+  :ref:`[Example] <sphx_glr_auto_examples_howto_plot_optimize_evoked.py>`
```

Comment:
```suggestion
- Add option to select drives using argument `which_drives` in :func:`~hnn_core.optimization.optimize_evoked`, by `Mohamed Sherif`_ in :gh:`478`.
```

and then down below add your name with a link. Here is a rendering of the page:

https://jonescompneurolab.github.io/hnn-core/dev/whats_new.html